picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
+        'albumartists': 'WM/ALBUMARTISTS',
```

Comment:
Similar question, what's the difference between "albumartist" and "albumartists"?